added advanced configuration when creating the Notebook in the AI Lab (#6483)
DaoCloud/DaoCloud-docs: docs/en/docs/baize/developer/notebooks/create.md

    ![click button](../../images/notebook01.png)

1. The system will pre-fill basic configuration data, including the cluster, namespace,
   queue, priority, resources, and job arguments. Adjust these arguments and click **OK**.

    ![fill forms](../../images/notebook02.png)

> 1. In the **Advanced** configuration, you can customize parameters such as task parameters, scheduling policies, task analysis, SSH access, and auto-saving.

Reviewer: ```suggestion
1. In the **Advanced** configuration, you can customize arguments, scheduling policies, analysis, SSH, and auto-saving.
```
